ambient.d.ts

```diff
@@ -468,10 +468,52 @@ declare module '@mojaloop/central-services-error-handling' {
 }
 declare module '@mojaloop/central-services-stream' {
   import { EventEmitter } from 'events';
-  import { EventActionEnum, EventTypeEnum } from '@mojaloop/central-services-shared';
-  export interface KafkaConsumerConfig {
-    eventType: EventTypeEnum,
-    eventAction: EventActionEnum,
+  export interface GenericMessage<T, U> {
+    value: {
+      from: string,
+      to: string,
+      id: string,
+      content: {
+        uriParams: unknown,
+        headers: {
+          'content-type': string,
+          date: string,
+          'fspiop-source': string,
+          'fspiop-destination': string
+          authorization?: string,
+          'content-length': string,
+          host: string,
+        },
+        payload: string
+      },
+      type: string,
+      metadata: {
+        correlationId: string,
+        event: {
+          type: T,
+          action: U,
+          createdAt: string,
+          state: {
+            status: string,
+            code: number,
+            description: string
+          },
+          id: string,
+          responseTo: string,
+        },
+        trace: unknown
+        "protocol.createdAt": number
+      }
+    },
+    size: number,
+    key: unknown,
```

Comment:
shouldn't `key` be a string?

Reply:
In all of my tests, it's been `null` - I'll add a note to the file